Jenkinsfile

```diff
@@ -0,0 +1,101 @@
+#!groovy
+
+timestamps {
+  ansiColor('xterm') {
+    node('node-small') {
+      stage('SETUP') {
+        deleteDir()
+        dir('scm'){
+          checkout scm
+        }
+        dir('solrHome') {
+          git branch: SOLR_HOME_BRANCH, url: SOLR_HOME_GIT_URL
+        }
+      }
+      stage('BUILD') {
+        dir('scm') {
+          // Build the docker image, push to aws
+          sh """
+            docker build --tag ${DOCKER_REGISTRY_ADDRESS}/${STORE_NAMESPACE}/ep-store:\$(jq -r .version package.json) \
+              --build-arg BUILD_DATE="\$(date --rfc-3339=seconds)" --build-arg VERSION=\$(jq -r .version package.json) \
+              --build-arg VCS_REF=\$(git rev-parse HEAD) -f ./docker/dev/Dockerfile .
+            eval "\$(aws ecr get-login --no-include-email)"
+            docker push ${DOCKER_REGISTRY_ADDRESS}/${STORE_NAMESPACE}/ep-store:\$(jq -r .version package.json)
```

Comment:
what do you think about ep-store-dev, since it's something that could potentially be used for more than tests? going forward, we'll likely want to call it ep-blueprint-dev. maybe we should make that change now.

Reply:
Yeah, `ep-store-dev` makes sense.